k8s/upgrades/README.md

```diff
@@ -4,6 +4,7 @@
 
 This document describes the steps for the following OpenEBS Upgrade paths:
 
+- Upgrade from 1.0.0 to any of 1.1.0, 1.2.0, 1.3.0, 1.4.0, 1.5.0, 1.6.0, 1.7.0, 1.8.0, 1.9.0, 1.10.0, 1.11.0
```

Comment:
I'm not sure if mentioning eleven upgrade path matrix is useful or confusing. Instead, can't we say all in single line? ;
```suggestion
- Upgrade from 1.0.0 or later to a newer release up to 1.11.0
```